Installation/include/CGAL/config.h

```diff
@@ -422,6 +422,22 @@ using std::max;
 #  define CGAL_DEPRECATED
 #endif
 
+// Macro to trigger deprecation warnings with a custom message
+#ifdef CGAL_NO_DEPRECATION_WARNINGS
+#  define CGAL_DEPRECATED_MSG
+#elif defined(__GNUC__) || __has_attribute(__deprecated__)
+#  if BOOST_GCC >= 40500
+#    define CGAL_DEPRECATED_MSG(msg) __attribute__ ((deprecated(msg)))
+#  else
+#    define CGAL_DEPRECATED_MSG(msg) __attribute__ ((deprecated))
+#  endif
+#elif defined (_MSC_VER) && (_MSC_VER > 1300)
```

Comment:
Recent Clang also has the syntax with the message, and even more.
http://clang.llvm.org/docs/AttributeReference.html#deprecated-gnu-deprecated

So line 429 it should be:
```C++
#  if BOOST_GCC >= 40500  || __has_attribute(__deprecated__)
```